playbook/app/pb_kits/playbook/pb_collapsible/child_kits/CollapsibleIcon.tsx

```diff
@@ -68,9 +69,10 @@ const CollapsibleIcon = ({
               className="icon_wrapper"
               key={icon ? showIcon(icon)[0] : "chevron-down"}
               onClick={(e) => handleIconClick(e)}
-              style={{ verticalAlign: "middle", color: color }}
+              style={{ verticalAlign: "middle"}}
           >
             <Icon
+                color={color}
```

Comment:
The error section is not getting the correct color! Maybe the token value is incorrect for the Icon color prop?